models/players.js

```diff
@@ -135,6 +144,19 @@ formatEUSoccerPlayers = function(response){
   return players;
 }
 
+formatMLBPlayers = function(response, team_id){
+  parseString(response, function (err, result) {
+    var str = result[Object.keys(result)[0]];
+        for (j=0; j < str.team.length; j++){ 
+          if (str.team[j].$.id == team_id){
```

Comment:
hmm. JQuery is still really shaky for me. 

It's not recognizing jquery on this page. getting 
ReferenceError: $ is not defined

How do I get the jquery on this page? only thing I've seen is put html script tags on an html page..
<script src="https://code.jquery.com/jquery-1.10.2.js"></script>